tools/cldr-code/src/main/java/org/unicode/cldr/draft/FileUtilities.java

```diff
@@ -279,7 +279,7 @@ public static BufferedReader openFile(String directory, String file, Charset cha
             if (directory.equals("")) {
                 return new BufferedReader(new InputStreamReader(new FileInputStream(new File(file)), charset));
             } else {
-                return new BufferedReader(new InputStreamReader(new FileInputStream(new File(file, directory)), charset));
+                return new BufferedReader(new InputStreamReader(new FileInputStream(new File(directory, file)), charset));
             }
```

Comment:
Optional: Pull the "if" and the File() construction out to make a local File variable, and then use that in a single, shorter return new BufferedReader(new ISR(... line.